src/sgame/sg_bot_util.cpp

```diff
@@ -1212,7 +1212,7 @@ bool BotTargetInAttackRange( const gentity_t *self, botTarget_t target )
 				range = glm::distance( muzzle, proj );
 
 				// make sure the sign of the range is correct
-				glm::vec3 rdir = glm::normalize( npos - muzzle );
+				glm::vec3 rdir = npos - muzzle;
```

Comment:
Why is this safe?  Is it because we only care about the sign of the dot product rather than the value? Let's comment about this.

Reply:
Yes exactly. I don't think it makes sense to add a comment in the code though. I guess I can put one in the commit but tbh someone discovering the code shouldn't be helped by that

Reply:
I personally do like these descriptions in the commit message at the very least. I helps people understand *why* certain changes were made. They are discoverable via blame too